Don't reimplement stuff that's already in glibc, musl and friends

diff -Naur make-4.4/Makefile.am make-4.4.tpg/Makefile.am
--- make-4.4/Makefile.am	2022-10-25 01:58:27.000000000 +0000
+++ make-4.4.tpg/Makefile.am	2022-11-02 11:16:14.676402669 +0000
@@ -30,8 +30,7 @@
 
 make_SRCS =	src/ar.c src/arscan.c src/commands.c src/commands.h \
 		src/debug.h src/default.c src/dep.h src/dir.c src/expand.c \
-		src/file.c src/filedef.h src/function.c src/getopt.c \
-		src/getopt.h src/getopt1.c src/gettext.h src/guile.c \
+		src/file.c src/filedef.h src/function.c src/guile.c \
 		src/hash.c src/hash.h src/implicit.c src/job.c src/job.h \
 		src/load.c src/loadapi.c src/main.c src/makeint.h src/misc.c \
 		src/os.h src/output.c src/output.h src/read.c src/remake.c \
@@ -68,8 +67,7 @@
   _GUILE_LIBS =
 endif
 
-make_LDADD =	$(LIBOBJS) $(_GUILE_LIBS) lib/libgnu.a $(GETLOADAVG_LIBS) \
-		@LIBINTL@
+make_LDADD =	$(LIBOBJS) $(_GUILE_LIBS) lib/libgnu.a $(GETLOADAVG_LIBS)
 
 localedir =	$(datadir)/locale
 
